pkg/config/config.go

```diff
@@ -33,6 +33,7 @@ const (
 	flagAWSEndpointURL       = "aws-endpoint-url"
 	flagLogLevel             = "log-level"
 	flagResourceTags         = "resource-tags"
+	flagWatchNamespace       = "watch-namespace"
```

Comment:
Can you make this `watch-namespaces` and allow for multiple string args please? (see how `flagResourceTags` is created...)

Reply:
Ignore this comment. I forgot upstream `ctrlrt.NewManager` call doesn't support >1 namespace.